### DISCUSSION TITLE
HttpMultipartForm class

### DISCUSSION THREAD

**Discussion by centratelemedia** [activej/activej#155] [General]
after expolore http core, i was not found class encoder for MultipartForm data, is there are sample for that?

**Comment by eduard-vasinskyi** [comment]
Hi, @centratelemedia

ActiveJ only includes a parser for MultipartForm data, not the encoder. If you wish, you can use your own MultipartForm data encoder to encode `HttpRequest`’s body or body stream.

**Reply by centratelemedia** [reply]
@eduard-vasinskyi 
Thank for answer...
